src/shared/lib/hooks/useChangeStep.ts

```diff
@@ -0,0 +1,20 @@
+import {useCallback} from "react";
```

Comment:
This hook is created for the reset-password form. I don't think it can be reused anywhere, so you need to move it to features/authentication/reset-password/hooks